container-images/mds-config/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+// Express local
+import { ApiServer } from '@mds-core/mds-api-server'
+import { api } from '@mds-core/mds-config'
+import { env } from '@container-images/env-inject'
+
+const { npm_package_name, npm_package_version, npm_package_git_commit, PORT = 4008 } = env()
```

Comment:
identity is using port `4008`. Do we anticipate any port conflicts with `lacuna-mds` and `mds-core` apis?

Reply:
I thought about that. My recommendation is to use a different `PORT` range, perhaps `5000-` for APIs that are not in `mds-core` .. so we can avoid these conflicts.

Reply:
@invertigo do you have thoughts about this? Using 5xxx in lacuna-mds apis would avoid port conflicts. If you're behind this, I'll try to update identity before it gets deployed.

Reply:
@levi217 I would think lacuna-mds should use a different range.  For apps we use a simple readme to track port assignments and we use 3000-